mern/client/src/screens/UploadTranscript/uploadTranscript.js

```diff
@@ -4,13 +4,11 @@ import { useNavigate } from "react-router-dom";
 import styles from "./uploadTranscript.module.css";
```

Comment:
TBH this file is pretty large just because the upload file use case has so many tasks. Many of the buttons onClicks call multiple different functions which then clutters the default function with a lot of code. Maybe these functions can be extracted. Especially since this file contacts functions for upload transcript and create flow. We can separate functions pertaining to each use case into different files.